### ISSUE TITLE
Setup IntHub and SciHub credentials in Secrets Manager

### ISSUE THREAD

**Issue by ciaransweet** [NASA-IMPACT/hls-sentinel2-downloader-serverless#3]
# TL;DR

The credentials used to query and download imagery via IntHub and SciHub need to be stored within Secrets Manager so that they can be accessed by the services which need them

![image](https://user-images.githubusercontent.com/9111975/102629636-4c152300-4143-11eb-9115-21988f16ce9a.png)

# Info

CDK only supports the creation of the secrets themselves, not the values of those secrets (That is to say that it will generate you a value, but that isn't much good if you've already registered a password with a 3rd party service).

AWS CDK documentation [says](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-secretsmanager-readme.html):

```
If you need to use a pre-existing secret, the recommended way is to manually provision the secret in AWS SecretsManager and use the Secret.fromSecretArn or Secret.fromSecretAttributes method to make it available in your CDK Application:
```

To that end, I'm recommending that we manually create the secrets and their values via the AWS console and then provide the ARNs as environment variables to our deployment, that way we can provide these to Lambda functions etc that need to access the secrets.

# Acceptance criteria

* IntHub Username and Password stored in Secrets Manager
* SciHub Username and Password stored in Secrets Manager
